llama2-7b-chat/README.md

```diff
@@ -110,6 +110,37 @@ for chunk in chat_completion:
     print(chunk.choices[0].delta.content or "", end="")
 ```
 
+These OpenAI-compatible endpoints also support [vLLM extra parameters](https://docs.vllm.ai/en/latest/serving/openai_compatible_server.html#extra-parameters). For example, you can force the chat completion output a JSON object by using the `guided_json` parameters:
```

Comment:
`guided_json` means `json_schema` here? Users don't need to set it in the `client.chat.completions.create` function?

Reply:
Yes, `guided_json` just forces the output to obey the JSON schema. User passed `guided_json` to extra_body. I missed that in my code, just updated.